app/test/test_langid.py

```diff
@@ -91,18 +91,8 @@ def test_langid_api_get(self):
         self.assertEqual('application/json', response.content_type)
         self.assertEqual(200, response.status_code)
 
-    def test_langid_api_get_with_query_request(self):
-        response = self.client.get(
-            '/text/langid/?text=Hello',
-        )
-        result = json.loads(response.data.decode())
-        self.assertEqual('en', result['result']['language'])
-        self.assertEqual(app.config['PROVIDER_LANGID'], result['provider'])
-        self.assertEqual('application/json', response.content_type)
-        self.assertEqual(200, response.status_code)
-
     def test_langid_api_get_without_text(self):
-        response = self.client.get(
+        response = self.client.post(
```

Comment:
looks like Alegre's tests are still doing get not post, maybe that will update later? https://github.com/meedan/alegre/blob/ef93a7547ab9d1952a171d50448509cafa0350c5/app/test/test_langid.py#L81

Reply:
I'm .... not seeing this reflected in the current state of this code? at least locally?